tests/netflix/test_time_series.py

```diff
@@ -0,0 +1,78 @@
+from service_capacity_modeling.capacity_planner import planner
+from service_capacity_modeling.interface import CapacityDesires
+from service_capacity_modeling.interface import DataShape
+from service_capacity_modeling.interface import Interval
+from service_capacity_modeling.interface import QueryPattern
+
+
+def test_timeseries_increasing_qps_simple():
+    qps_values = (100, 1000, 10_000, 100_000)
```

Comment:
Since this is a pretty slow test to run on every model we could just have 1k, 10k, 100k

Reply:
Updated